ChangeLog.md

```diff
@@ -1,3 +1,18 @@
+# wolfSSL Release 4.4.0 (To Be Determined / Work in Progress)
```

Comment:
Should we do X.X.X, instead of 4.4.0, in case we push out a minor version instead (ie: 4.3.2)?  Also applies a few lines down.